# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mmtisobmff/remove-submodules.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mmtisobmff-r1.0.1/CMakeLists.txt.vanilla	2025-09-24 19:18:22.731010524 +0200
+++ mmtisobmff-r1.0.1/CMakeLists.txt	2025-09-24 19:18:33.800253863 +0200
@@ -1,16 +1,7 @@
 cmake_minimum_required( VERSION 3.16.0 )
-include(FetchContent)
 
 project(mmtisobmff VERSION 1.0.1)
 
-FetchContent_Declare(
-  ilo
-  GIT_REPOSITORY https://github.com/Fraunhofer-IIS/ilo.git
-  GIT_TAG        r1.0.0
-)
-
-FetchContent_MakeAvailable(ilo)
-
 if(mmtisobmff_included)
   return()
 endif(mmtisobmff_included)
